db/client.js

```diff
@@ -1,11 +1,18 @@
-const { Client } = require("pg");
+// const { Client } = require("pg");
```

Comment:
Perhaps undo your changes to the client.js file, since we want Sean's changes to be the version we use.